react-components/src/components/RevealToolbar/RevealToolbar.tsx

```diff
@@ -52,15 +54,27 @@ const DefaultContentWrapper = (props: RevealToolbarProps): ReactElement => {
   );
 };
 
-const RevealToolbarContainer = (
-  props: RevealToolbarProps & { toolBarContent?: JSX.Element }
-): ReactElement => {
-  if (props.className === undefined && props.style === undefined) {
-    props = { ...props, ...defaultStyle };
+const RevealToolbarContainer = ({
+  customSettingsContent,
+  lowFidelitySettings,
+  highFidelitySettings,
+  toolBarContent,
+  ...rest
```

Comment:
Can we write `restProps` or something similar just to indicate that it's props we're talking about?